conf/params.config

```diff
@@ -0,0 +1,13 @@
+//standard configs for all the processes
+
+params {
+    input_type = "reads"
+    outdir = "${baseDir}/wgs_output"
+    save_mode = "copy"
+    should_publish = "true"
+    USER = "root"
+
+    //mtbseq
+    mtbseq_library_name = "illumina"
```

Comment:
@Mxrcon , for the time being let's just leave it as `somelib` since the analysis which was done earlier with that configuration.

Reply:
ok! I'll update that!